NeoMathEngine/src/CPU/CpuMathEngineDnnConv.cpp

```diff
@@ -358,10 +347,7 @@ void CCpuMathEngine::blobConvolutionForwardAlgo1( const CCpuConvolutionDesc& des
 	const int outputTransposedDataObjectSize = result.Width() * outputTransposedDataRowSize;
 	const int tempBlobDataRowSize = result.Height() * filter.Height() * filter.Width() * source.Depth() * source.Channels();
 	const int tempBlobDataObjectSize = result.Width() * tempBlobDataRowSize;
-
-	const int curThreadCount = IsOmpRelevant( source.ObjectCount() * result.Width(),
-		static_cast<int64_t>( source.BlobSize() ) * filter.BlobSize() ) ? threadCount : 1;
-	const int tempObjectCount = std::min( source.ObjectCount(), curThreadCount );
+	const int tempObjectCount = source.ObjectCount();
```

Comment:
`std::min( source.ObjectCount(), curThreadCount );` is `1`, not `source.ObjectCount();`

Reply:
fixed